config/environments/sandbox.rb

```diff
@@ -81,7 +81,7 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
-
+  
```

Comment:
```suggestion
```

Reply:
never mind. I think if I don't click 'Add suggestion' to batch, this change will be applied right?

Reply:
So I can just resolve the conversation?